docs/sources/contribute/documentation.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Documentation Guidelines"
+description = "Documentation Guidelines"
+type = "docs"
+[menu.docs]
+parent = "contribute"
+weight = 2
++++
+
+# Documentation Guidelines
```

Comment:
I suggest a different title. This title does not match the content. Maybe: Contribute to Grafana documentation